src/main/java/graphql/parser/GraphqlAntlrToLanguage.java

```diff
@@ -688,9 +691,12 @@ private Value getValue(GraphqlParser.ValueContext ctx) {
         throw new ShouldNotHappenException();
     }
 
-    private String parseString(String string) {
+    static String parseString(String string) {
+        assert string.length() >= 2;
```

Comment:
We don't use assert statements in prod code: they are turned off by default and don't have any effect.

Also it is not really needed to ensure the length or that the string starts wit `"`, this is already ensured by the parser/lexer.